README.md

```diff
@@ -3,23 +3,30 @@
 Why use it: trusted implementations, novel metrics, performance, one-stop-shop.
 
+Included Metrics:
+
+| continuous | probability | statistical tests      |
+| ---------- | ----------- | ----------- |
+| MAE, MSE, RMSE, FIRM | CRPS, FIRM |  Diebold Mariano (with the Harvey 1997 and Hering 2011 modifications) |
+
+
 **Notice -- This repository is currently undergoing initial construction and maintenance. It is not yet recommended for use. This notice will be removed after the first feature release. In the meantime, please feel free to look around, and don't hesitate to get in touch with any questions (see the contributing guide for how).**
 
-This package is currently in very active development. An additional ten to fifteen scores are expected to be implemented in the coming weeks. Performance testing and dask compatibility will be reviewed during and afterwards. The first 'release' will be tagged at that point.
+Documentation is hosted at [scores.readthedocs.io](https://scores.readthedocs.io)
 
-The documentation is divided into the [user guide](docs/userguide.md), the [contribution guide](docs/contributing.md) (including developer documentation) and [API documentation](docs/api.md).
+`scores` is a modular scoring package containing verification metrics, error functions, training scores and other statistical functions. It primarily supports the geoscience and earth system science communities. `scores` is focused on supporting xarray datatypes for earth system data. It has wide potential application in machine learning, and domains other than meteorology, geoscience and weather. It also aims to be compatible with pandas, geopandas, pangeo and work with NetCDF4, Zarr, hdf5 and GRIB data sources among others.
 
-`scores` is a modular scoring package containing mathematical functions that can be used for the verification, evaluation, and optimisation of model predictions, as well as other statistical functions. It is primarily aiming to support the geoscience and earth system community. `scores` is focused on supporting xarray datatypes for earth system data. Other data formats such as Pandas and Iris can be easily be converted to xarray objects to utilise `scores`. It has wider potential application in machine learning and domains other than meteorology, geoscience and weather but primarily supports those fields. It aims to be compatible with geopandas, pangeo and work with NetCDF4, Zarr, and hd5 data sources among others. To use `scores` with GRIB data, install [cfgrib](https://github.com/ecmwf/cfgrib) into your python environment and use `engine='cfgrib'` when opening a grib file with xarray.
+`scores` includes novel scores not commonly found elsewhere (e.g. FIRM and FlipFlop Index), complex scores (CRPS, Diebold Mariano) and more common scores (MAE, RMSE). `scores` provides its own implementations where relevant to avoid extensive dependencies.
```

Comment:
```suggestion
`scores` includes novel scores not commonly found elsewhere (e.g. FIRM and FlipFlop Index), complex scores (CRPS) and more common scores (MAE, RMSE). `scores` provides its own implementations where relevant to avoid extensive dependencies.
```
I suggest removing Diebold Mariano since it's a statistical test rather than a verification score (or explicitly calling it a statistical test rather than lumping it with CRPS.